halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/dockerRegistry/DockerRegistryEditAccountCommand.java

```diff
@@ -88,6 +88,12 @@ protected String getProviderName() {
   )
   private Long cacheIntervalSeconds;
 
+  @Parameter(
+      names = "--insecure-registry",
+      description = DockerRegistryCommandProperties.INSECURE_REGISTRY_DESCRIPTION
```

Comment:
Can you make the arity=1 here? This allows you to set the flag back to false.

Reply:
I've set arity=1 for both the add and edit command.  Were you also suggesting setting the default value of the flag to false?  I intentionally left that default null in the edit command for the isSet() check.